CONTRIBUTING.md

```diff
@@ -202,3 +202,18 @@ New dependency: dependency_name
 license: SPDX license tag
 origin: https://dependency_origin.com
 ```
+## Code coverage
```

Comment:
please add an empty line before this section

Reply:
Done